# Define the request
float64[] joint_positions
---
# Define the result
hexapod_control/Pose solution
int32 result
string error_message